src/rev_ai/apiclient.py

```diff
@@ -178,26 +178,36 @@ def get_list_of_jobs(self, limit=None, starting_after=None):
 
         return [Job.from_json(job) for job in response.json()]
 
-    def get_transcript_text(self, id_):
+    def get_transcript_text(self, id_, filename = None, filepath = ''):
```

Comment:
Is there a reason that these default to different things? Defaulting them both to `None` might make more sense